Automattic/jetpack
# In Jetpack_Google_Analytics, use the static property for instance of Universal or Legacy analytics

Fixes https://github.com/Automattic/jetpack/issues/19755

#### Changes proposed in this Pull Request:
* Updated the `__construct` function in `Jetpack_Google_Analytics` to use `self::$analytics` rather than `$analytics` so that the instance of either `Jetpack_Google_Analytics_Universal` or `Jetpack_Google_Analytics_Legacy` is stored in the static property `analytics`.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:
* Enable Google Analytics
* Using the global instance of `$jetpack_google_analytics` ensure that `$jetpack_google_analytics::$analytics` is an instance of one of the analytics classes rather than false.
